lib/discourse_post_event/event_validator.rb

```diff
@@ -93,7 +93,11 @@ def can_invite_groups?(event)
       return true unless event[:"allowed-groups"]
 
       event[:"allowed-groups"].split(',').each do |group_name|
-        group = Group.find_by(name: group_name)
+        group = begin
+          Group.lookup_group(group_name.to_sym)
```

Comment:
this api in core is a bit awkward, we should just do the simbolizing in core and keep the api as a string api... nothing to change here... but it is certainly odd.

Reply:
Yeah I thought the same.